src/lib/stitching/gravity/stitching.ts

```diff
@@ -166,6 +172,23 @@ export const gravityStitchingEnvironment = (
             })
           },
         },
+        addressConnection: {
+          fragment: gql`
+          ... on Me {
+            __typename
+          }
+          `,
+          resolve: (_parent, args, context, info) => {
+            return info.mergeInfo.delegateToSchema({
+              schema: gravitySchema,
+              operation: "query",
+              fieldName: "_unused_gravity_userAddressConnection",
+              args: { ...args, userId: context.userID },
```

Comment:
@zephraph, @sepans and I were wondering if there is a best practice here to use `userId` from context (which we're assuming we'll always have access to on `Me`) or to query for `userId` in the above fragment and in this line, use `parent.userId` instead in `args`. Wondering about best practice or performance considerations?

Reply:
A good question. For the performance, performing the extra query is likely negligible. In many cases it's likely the user whose querying this value will be querying other user data from `Me` anyway in which case the initial call would happen either way. 

I think how you have it is fine.